lms/urls.py

```diff
@@ -181,19 +181,22 @@
     from course_wiki import views as course_wiki_views
     from django_notify.urls import get_pattern as notify_pattern
 
+    wiki_url_patterns, wiki_app_name, wiki_namespace = wiki_pattern()
+    notify_url_patterns, notify_app_name, notify_namespace = notify_pattern()
+
     urlpatterns += [
         # First we include views from course_wiki that we use to override the default views.
         # They come first in the urlpatterns so they get resolved first
         url('^wiki/create-root/$', course_wiki_views.root_create, name='root_create'),
-        url(r'^wiki/', include(wiki_pattern())),
-        url(r'^notify/', include(notify_pattern())),
+        url(r'^wiki/', include((wiki_url_patterns, wiki_app_name), namespace=wiki_namespace)),
+        url(r'^notify/', include((notify_url_patterns, notify_app_name), namespace=notify_namespace)),
 
         # These urls are for viewing the wiki in the context of a course. They should
         # never be returned by a reverse() so they come after the other url patterns
         url(r'^courses/{}/course_wiki/?$'.format(settings.COURSE_ID_PATTERN),
             course_wiki_views.course_wiki_redirect, name='course_wiki'),
         url(r'^courses/{}/wiki/'.format(settings.COURSE_KEY_REGEX),
-            include(wiki_pattern(app_name='course_wiki_do_not_reverse', namespace='course_wiki_do_not_reverse'))),
+            wiki_pattern(app_name='course_wiki_do_not_reverse', namespace='course_wiki_do_not_reverse')),
```

Comment:
This one should also be updated to still use `include()`.  After that this should be good to merge